### PR TITLE
Fix for issue #551 

### PR DESCRIPTION
**Pull request by mzpqnxow**
# Description

Please see #551 which explains the issue and the solution. This is a simple change that does not require QA or dedicated testing in my view

### CODE REVIEW

File: tenable/reports/nessusv2.py [hunk 1]
```diff
@@ -54,6 +54,8 @@ def _defs(self, name, value):
 
         elif name in ['cvss_base_score', 'cvss_temporal_score']:
             # CVSS scores are floats, so lets return them as such.
+            if value is None:  # Issue #551
+                return 0
             return float(value)
```
Comment (SteveMcGrath) [inline tenable/reports/nessusv2.py:59]:
```suggestion
            if value:
                return float(value)
```

**Comment by SteveMcGrath** [comment]
Sorry for the delay.  I was waiting for the engineering team that took this over to comment and take the appropriate action.

File: tenable/reports/nessusv2.py (hunk 1 above)
Comment (mzpqnxow) [reply tenable/reports/nessusv2.py:59]: > Sorry for the delay. I was waiting for the engineering team that took this over to comment and take the appropriate action.

No worries, I appreciate you taking the time to review. Also, I agree with that proposed behavior

**Comment by mzpqnxow** [comment]
I "committed" the recommendation- I didn't even realize you could  do that with GitHub, very convenient :)

**Pull request merged**